docs/releases/release-1.0.md

```diff
@@ -0,0 +1,56 @@
+is, the major version of Kubeflow Pipelines is incremented for API/feature changes that
+are new and not backward-compatible; and on the other hand, the minor version
+of Kubeflow Pipelines is incremented for API/feature changes that are new and
+backward-compatible.
+
+## Deployment options
+
+* [Standalone Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/installation/standalone-deployment/),
+which allows users the maximum flexibility in configuring the Kubernetes
+clusters and a Kubeflow Pipelines instance.
+
+* [Google Cloud Marketplace Hosted Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/installation/overview/#gcp-hosted-ml-pipelines),
+which minimizes the user effort in deploying a Kubeflow pipelines instance.
+
+## Core features and APIs
+
+Core features and APIs are the fundamental functionalities of Kubeflow Pipelines. The most fundamental functionality is to manage the Kubeflow Pipelines resources (that is, pipeline, pipeline version, run, job, experiment) via the operations listed below. Active supports are available for core features and APIs.
+
+|                  | Upload | Create | Get | List | Delete | Archive | Unarchive | Enable | Disable | Terminate | Retry |
+|:----------------:|:------:|:------:|:---:|:----:|:------:|:-------:|:---------:|:------:|:-------:|:---------:|:-----:|
+| Pipeline         | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |
+| Pipeline Version | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |
+| Run              | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |        |         | - [x]     | - [x] |
+| Job              | - [x]  |  - [x] |- [x]|- [x] | - [x]  |         |           | - [x]  | - [x]   |
+| Experiment       | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |
+
+* Refer to the Kubeflow Pipelines [introduction]((https://www.kubeflow.org/docs/pipelines/overview/)) for
+  - An [overview](https://www.kubeflow.org/docs/pipelines/overview/pipelines-overview/) and [main concepts](https://www.kubeflow.org/docs/pipelines/overview/concepts/), for example, what is a pipeline or run in Kubeflow Pipelines.
+  - How to manage the Kubeflow Pipelines resources via the the [Kubeflow Pipelines UI](https://www.kubeflow.org/docs/pipelines/overview/interfaces/).
+* Refer to the Kubeflow Pipelines [SDK samples and tutorials](https://www.kubeflow.org/docs/pipelines/tutorials/sdk-examples/)
+for how to manage the Kubeflow Pipelines resources via the Kubeflow Pipelines client.
+* A complete API method list and a detailed description of parameters to those API methods are available at the Kubeflow Pipelines [API reference](https://www.kubeflow.org/docs/pipelines/reference/api/kubeflow-pipeline-api-spec/).
+
+## Experimental features
+
+Experimental features are the enhanced functionalities available in Kubeflow Pipelines. They might be replaced or deprecated in the future due to the refactoring or redesign of Kubeflow Pipelines.
```

Comment:
Maybe we can borrow the description for Kubeflow alpha applications:
> Alpha means that the application is in the early phases of development and/or integration into Kubeflow.

We may rephrase like
```
Experimental means that the feature is in the early phases of development and may change drastically in the future.
```

Reply:
Another good source of reference is Kubernetes:
* https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-versioning
* https://kubernetes.io/docs/reference/using-api/deprecation-policy/

Reply:
Ah for the api versioning, we'll probably need to revise the v1beta in the swagger json

Reply:
Agree, that's quite some piece of work changing api prefix. I'd prefer we do that after 1.0 public release, what do you think?